google-cloud-spanner/acceptance/spanner/client/types/numeric_test.rb

```diff
@@ -172,7 +172,7 @@
   end
 
   it "writes and queries primary key as a numeric" do
-    skip if emulator_enabled?
+    skip # if emulator_enabled?
```

Comment:
Should we remove this change from this PR? Because we will merge the fix first and then let this PR rebase with the main branch.